vendor.conf

```diff
@@ -17,4 +17,7 @@ github.com/tchap/go-patricia v2.2.6
 github.com/vbatts/tar-split bd4c5d64c3e9297f410025a3b1bd0c58f659e721
 github.com/vdemeester/shakers 24d7f1d6a71aa5d9cbe7390e4afb66b7eef9e1b3
```

Comment:
This PR removes the module and `github.com/go-check/check`.  Remove them while we're in here?